homeassistant/components/yamaha/media_player.py

```diff
@@ -99,14 +101,16 @@
 class YamahaConfigInfo:
     """Configuration Info for Yamaha Receivers."""
 
-    def __init__(self, config: ConfigType, discovery_info: DiscoveryInfoType) -> None:
+    def __init__(
+        self, config: ConfigType, discovery_info: DiscoveryInfoType | None
+    ) -> None:
         """Initialize the Configuration Info for Yamaha Receiver."""
         self.name = config.get(CONF_NAME)
         self.host = config.get(CONF_HOST)
         self.ctrl_url: str | None = f"http://{self.host}:80/YamahaRemoteControl/ctrl"
         self.source_ignore = config.get(CONF_SOURCE_IGNORE)
         self.source_names = config.get(CONF_SOURCE_NAMES)
-        self.zone_ignore = config.get(CONF_ZONE_IGNORE)
+        self.zone_ignore = config.get(CONF_ZONE_IGNORE, [])
```

Comment:
This should be handled automatically be the schema validation, but it seemed easier to change it here than to adjust the tests. I'm happy to adjust the tests instead if so desired.

Reply:
Well, in this location, it isn't correct. So, we should adjust the tests in that case.

Reply:
I struggled a bit with that one as I am not so familiar with the test.
I think it's good now.